packages/expo-video/android/src/main/java/expo/modules/video/IntervalUpdateClock.kt

```diff
@@ -0,0 +1,54 @@
+package expo.modules.video
+
+import android.os.Handler
+import android.os.Looper
+import expo.modules.video.delegates.IgnoreSameSet
+import java.lang.ref.WeakReference
+
+interface IntervalUpdateEmitter {
```

Comment:
```suggestion
fun interface IntervalUpdateEmitter {
```
It can be a functional interface.